src/main/java/com/google/sps/servlets/LogInServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Transaction;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+// import com.google.appengine.api.oauth.OAuthService;
+// import com.google.appengine.api.oauth.OAuthServiceFactory;
+// import com.google.appengine.api.users.User;
```

Comment:
Please delete them if they will not be used yet.

Reply:
Done